src/app/common/directive/sort-course-directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, ElementRef, OnInit, Input } from '@angular/core';
+
+@Directive({
+  selector: '[sortCourseDirective]'
+})
+export class SortCourseDirective implements OnInit {
+  @Input() creationdate: Date;
+  currentElement: ElementRef;
+  constructor(private el: ElementRef) {
+    this.currentElement = el;
+  }
+
+  ngOnInit() {
+    const creationdate = new Date(this.creationdate);
+    const currentDate = new Date();
+    const previousDate = new Date();
+    const backDate = new Date(previousDate.setDate(currentDate.getDate() - 14));
+    if(creationdate < currentDate && creationdate >=backDate)
```

Comment:
Please use spaces and curly braces to succeed all linting rules

Reply:
Updated